src/style.css

```diff
@@ -1,3 +1,68 @@
-body {
+*,
+*::before,
+*::after {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  /* width: 1024px; */
+
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  width: 100%;
+  display: flex;
+  justify-content: right;
+
+}
+
+.nav__ul {
+  list-style: none;
+  display: flex;
+}
+
+.nav__li {
+  text-transform: uppercase;
+}
+
+.nav__link {
+  text-decoration: none;
+  margin-left: 20px;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  color: #000;
+  display: flex;
+  align-items: center;
+  height: 60px;
```

Comment:
better to set vertical padding instead of fixed height

Reply:
Padding to the links? Then total height will be more then 60px